xbmc/music/GUIViewStateMusic.cpp

```diff
@@ -280,6 +282,7 @@ CGUIViewStateMusicDatabase::CGUIViewStateMusicDatabase(const CFileItemList& item
       AddSortMethod(SortByRating, 563, LABEL_MASKS("%T - %A", "%R"));  // Title - Artist, Rating
       AddSortMethod(SortByYear, 562, LABEL_MASKS("%T - %A", "%Y")); // Title, Artist, Year
       AddSortMethod(SortByDateAdded, 570, LABEL_MASKS("%A - %T", "%a"));  // Title - Artist, DateAdded | empty, empty
```

Comment:
All `SortByFoo` methods here use `%T - %A` as the label format but `SortByDataAdded` uses `%A - %T`. It's not part of this PR but I just noticed that it's inconsistent.